src/ws-notification/ws-notification.js

```diff
@@ -94,7 +95,7 @@ export class WSNotification extends Component {
    * @returns {void}
    */
   animateIn(notification, index) {
-    const list = this.list;
+    const {list} = this.list;
```

Comment:
Just recognized while fixing the tons of merge conflicts. const list = this.list can not be replaced by a destruction in this way